.github/workflows/node.js.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: ['18.x', '20.x', '22.x']
```

Comment:
do you think it's worth it to test multiple versions of node? feel like just one would work fine (whatever spectral recommends at the moment).

Reply:
I've seen issues in other projects like "Failure with Node 22.0". I think this gives us some protection against this kind of problem. And I *think* it's cheap -- the tests run very quickly -- so let's keep it for now.